ophyd/pseudopos.py

```diff
@@ -126,11 +334,14 @@ class PseudoPositioner(Device, Positioner):
     '''
     def __init__(self, prefix, *, concurrent=True, read_attrs=None,
                  configuration_attrs=None, monitor_attrs=None, name=None,
-                 **kwargs):
+                 egu='', **kwargs):
```

Comment:
`PseudoPositioner` egu, if unspecified, could default to being a composite of its `PseudoSingle` egus?